OrbitGrpcProtos/capture.proto

```diff
@@ -45,29 +45,43 @@ message CaptureOptions {
   bool enable_introspection = 9;
 }
 
+// For CaptureEvents with a duration, excluding for now GPU-related ones, we
```

Comment:
why excluding GPU-related ones?

Reply:
A couple of reasons:
- I don't want to touch the events coming from the Vulkan layer and interfere with Florian's work.
- As for GPU jobs, Florian is also working with them. Also, they have three durations and I didn't want to deal with that for now.
As they are low frequency compared to other events, I just let them alone.